docs/spanner/usage.rst

```diff
@@ -1,11 +1,21 @@
 Spanner
-=======
+^^^^^^^
+
+`Cloud Spanner`_ is the world’s first fully managed relational database service
```

Comment:
You have a non-ascii single quote here, which I assume is an accident

Reply:
Yep, I copy and pasted that from cloud site.